### PR TITLE
feature/#754 improve config schema

### PR DESCRIPTION
**Pull request by toan-quach**
#754

### CODE REVIEW

File: src/taipy/core/config/config.schema.json [hunk 1]
```diff
@@ -8,7 +8,9 @@
     "CORE": {
       "description": "Data nodes",
       "type": "object",
-      "properties": {},
+      "properties": {
+        "version": "3.0.0"
```
Comment (jrobinAV) [inline src/taipy/core/config/config.schema.json:12]: should we have a description and a type instead of a value ?

Comment (toan-quach) [reply src/taipy/core/config/config.schema.json:12]: @jrobinAV do you mean something like: 
```suggestion
        {"version": 
                {"description": "The version of Taipy Core",
                "type": "string"
                }
        }
```

Comment (jrobinAV) [reply src/taipy/core/config/config.schema.json:12]: Yes, but @FredLL-Avaiga should know better.

**Comment by jrobinAV** [comment]
Is this PR closed on purpose?

File: src/taipy/core/config/config.schema.json (hunk 1 above)
Comment (FredLL-Avaiga) [reply src/taipy/core/config/config.schema.json:12]: yes that would do.
What about the sequences description ?

**Comment by toan-quach** [comment]
@jrobinAV @FredLL-Avaiga this PR has been closed and will be moved over to PR https://github.com/Avaiga/taipy-core/pull/762 so the topic is easier to maintain and follow

**Pull request closed without merging**